cms/templates/certificate_page.html

```diff
@@ -1,46 +1,137 @@
 {% extends "base.html" %}
 {% load static wagtailimages_tags %}
 
+{% block title %}{{ site_name }} | Course Certificate{% endblock %}
+
+{% block seohead %}
+    {{ block.super }}
+    <meta property="og:site_name" content="{{ site_name }}" />
+    <meta property="og:type" content="website" />
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta name="twitter:site" content="@MITxonedX" />
+    <meta name="twitter:image" content="{{ share_image_url }}" />
+    <meta property="og:title" content="{{ site_name }} | Course Certificate: {{ page.product_name }}" />
+    <meta property="og:description" content="Course Certificate for {{ page.product_name }} awarded by {{ site_name }}." />
+    <meta property="og:url" content="{{ request.build_absolute_uri }}" />
+    <meta property="og:image:url" content="{{ share_image_url }}" />
+    <meta property="og:image:width" content="{{ share_image_width }}" />
+    <meta property="og:image:height" content="{{ share_image_height }}" />
+    <meta property="og:image:alt" content="A certificate from MIT xPRO" />
+{% endblock %}
+
 {% block content %}
-<div class="certificate-wrapper">
-    <div class="certificate">
-        <div class="certificate-holder">
-            <div class="certificate-logo">
-                <img src="{% static 'images/certificates/certificate-mit-logo.png' %}" alt="">
+<div class="container-fluid certificate-page">
+    <div class="row no-print">
+        <div class="col px-0">
+            <div class="cer-head">
+                <div class="certificate-logo">
+                    <a href="https://web.mit.edu/" class="mit"></a>
+                    <a href="/" class="xpro"></a>
+                    <img src="{% static 'images/certificates/transparent-mit-xpro-logo.png' %}" alt="">
+                </div>
             </div>
-            <span class="institute-text">Massachusetts Institute of Technology</span>
-            <div class="institute-logo">
-                <img src="{% static 'images/certificates/certificate-logo.png' %}" alt="">
+        </div>
+    </div>
+    {% if certificate_user == user %}
+        <div class="row no-print">
+            <div class="col px-0">
+                <div class="cer-user-info">
+                    <div class="user-info-holder">
+                        <ul class="social-links">
+                        <li>
+                            <a href="https://twitter.com/intent/tweet?url={{ request.build_absolute_uri|urlencode }}&text=I%20just%20earned%20a%20certificate%20in%20{{ page.product_name|urlencode }}%20from%20{{ site_name|urlencode }}">
```

Comment:
suggestion: Twitter `share text` should not be configurable too ? but I am seeing some characters in the string.

Reply:
Updated